src/InstallationValidator.Core/InstallationValidatorConfiguration.cs

```diff
@@ -17,69 +16,46 @@ public interface IInstallationValidatorConfiguration : IApplicationConfiguration
       string PKSimBatchToolPath { get; }
       string PKSimBinaryExecutablePath { get; }
       string MoBiBinaryExecutablePath { get; }
-      string DimensionFilePath { get; set; }
       string DefaultOutputPath { get; }
    }
 
    public class InstallationValidatorConfiguration : OSPSuiteConfiguration, IInstallationValidatorConfiguration
    {
-      public InstallationValidatorConfiguration()
```

Comment:
Most of the code here has been moved to OSPSUiteConfiguration so I had to make some changes. Tested locally and it runs ok